content/install-ubuntu.md

```diff
@@ -55,137 +53,143 @@ Ubuntu may need a special option enabled to install and boot from Ubuntu until t
+Once Ubuntu starts and you have selected your language, any accessibility features that you may need, your keyboard selection, and network connectivity you will be asked to **Try Ubuntu without installing** or to **Install Ubuntu**. If you are attempting to restore a broken operating system, please choose **Try Ubuntu**.
 
->**IMPORTANT NOTE:** System76 recommends [backing up all important files from your computer](https://support.system76.com/articles/backup-files). The restore process is designed to leave your files intact, but it's best to be safe in case something goes wrong. Next, double-click on the **Install Ubuntu** icon on the desktop.
+>**IMPORTANT NOTE:** System76 recommends [backing up all important files from your computer](https://support.system76.com/articles/backup-files). The Ubuntu installation process will wipe any data on the drive that is selected for the installation. Next, double-click on the **Install Ubuntu** icon on the desktop.
 
-![Try or Install](/images/install-ubuntu/ubuntu-install-try-install.png)
+![First page of the installer](/images/install-ubuntu/install-ubuntu-24.04-1.png)
 
->**NOTE:** You may or may not see the screen pictured above. Your system may boot directly to the screen pictured below:
+>**NOTE:** You may or may not see the screen pictured above. Your system may boot directly to the screen pictured above:
 
-![Try or Install 2](/images/install-ubuntu/install-ubuntu-21.04-1.png)
+### Language and Keyboard layout
+
+![Keyboard Layout](/images/install-ubuntu/install-ubuntu-24.04-3.png)
+
+If all of your files are backed up, or if you are installing from scratch, pick the **Install Ubuntu** option to continue with the installation.
+
+![Try or Install](/images/install-ubuntu/install-ubuntu-24.04-5.png)
+
+For most installations the **Interactive installation** type is recommended.
+
+![Type of Installation](/images/install-ubuntu/install-ubuntu-24.04-6.png)
+
+The **Default selection** will work for most installations while the **Extended selection** includes applications such as the LibreOffice suite and more. Those applications can be installed once the installation is finished if the **Default selection** is used instead.
 
-If all of your files are backed up, or if you are installing from scratch, pick the **Install Ubuntu** option to continue with the installation. Please check the box labeled **Download updates while installing Ubuntu** to ensure that your new installation is fully up to date once the installation is complete.
+![Applications](/images/install-ubuntu/install-ubuntu-24.04-7.png)
 
-![Download Updates](/images/install-ubuntu/ubuntu-install-screen2.png)
+Let's break down what each of the options offer in regards to hardware and software support:
+
+- **Install third-party software for graphics and Wi-Fi hardware**
+
+This installs drivers (some non-free) for support for Wi-Fi (Wireless) and GPU support mainly NVIDIA as the amdgpu driver for AMD GPU's is in the kernel.
```

Comment:
This is a run-on sentence. A comma is needed before "mainly," and would also be needed before your use of "as" to mean "because." I'd recommend using parenthesis so you don't need so many commas right next to each other.

"Wireless" should not be capitalized in the middle of a sentence.

AMDGPU should be all-caps rather than all-lowercase in this context. Let's rephrase "AMD GPU's" so we don't need to worry about whether that apostrophe is correct or not.

I'd also clarify "in the kernel" by adding the word "included," since a user who needs this article may not know what that means intuitively.

Finally, rephrase "for support for Wi-Fi" so you're not repeating "for" so close together.

> This installs drivers (some non-free) to provide support for Wi-Fi (wireless) and GPU support (mainly NVIDIA, as the AMDGPU driver for AMD graphics is included in the kernel).

Reply:
Actually, you shouldn't be repeating the word "support: so close together, either.

> This installs drivers (some non-free) to provide Wi-Fi (wireless) and GPU support (mainly NVIDIA, as the AMDGPU driver for AMD graphics is included in the kernel).

Reply:
If you're going to call out AMD here, why aren't you also calling out Intel?

Maybe simplify even further to:

> This installs drivers (some non-free) to provide Wi-Fi/Bluetooth and GPU support (mainly for NVIDIA graphics, as AMD and Intel graphics are supported natively by the Linux kernel).

Everyone knows that Wi-Fi means wireless, and nobody reading this article is likely to care what the AMDGPU module is called while performing this installation process.